examples/tutorials/Cell_Counting_Tutorial.ipynb

```diff
@@ -0,0 +1,503 @@
+{
```

Comment:
<p>Introduce the dataset. Make a new section. What is this dataset? Add a citation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3845/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
Done. Please review